exercises/pangram/example.rs

```diff
@@ -0,0 +1,18 @@
+use std::collections::BTreeSet;
+use std::ascii::AsciiExt;
+use std::iter::FromIterator;
+
+pub fn is_pangram(sentence: &str) -> bool {
+    sentence.to_lowercase()
+            .chars()
```

Comment:
Weird. The doc site has nothing for that function.

http://doc.rust-lang.org/std/?search=grapheme

And it doesn't seem to work. 

 https://play.rust-lang.org/?gist=02e6b2d55eb675a1a7e6c0bdae90454d&version=stable&backtrace=0

I've [asked Steve about it](https://twitter.com/iwhitney/status/738861168077348865)